### PR TITLE
fix build command

### PR DESCRIPTION
**Pull request by aryszka**
fix build command

Signed-off-by: Arpad Ryszka <arpad.ryszka@gmail.com>

### CODE REVIEW

**Comment by aryszka** [comment]
:+1:

File: packaging/Makefile [hunk 1]
```diff
@@ -34,9 +34,9 @@ docker-build: docker.build.amd64 docker.build.arm64 docker.build.armv7
 docker.build.amd64: clean skipper eskip
 	docker build -t $(IMAGE) .
 docker.build.arm64: clean build.linux.arm64 docker.build.enable
-	docker build build -t $(ARM64_IMAGE) --platform linux/arm64 -f Dockerfile.arm64 .
+	docker build -t $(ARM64_IMAGE) --platform linux/arm64 -f Dockerfile.arm64 .
```
Comment (szuecs) [inline packaging/Makefile:37]: `docker buildx build`

Comment (szuecs) [reply packaging/Makefile:37]: sorry I did the  `:%s/buildx/build/g`, which is wrong.
`docker buildx build` is correct.

**Comment by aryszka** [comment]
Not sure if this is enough. When ran it locally, the next error was:

```
"--platform" is only supported on a Docker daemon with experimental features enabled
```

despite having `{experimental: "enabled"}`. I had to set `{experimental: "true"}` in `/etc/docker/daemon.json` and restart the daemon.

After that, the next error was:

```
standard_init_linux.go:211: exec user process caused "exec format error"
```

At this point, I had to pause the investigation, due to other stuff.

**Comment by szuecs** [comment]
close in favor of #1413

**Pull request closed without merging**